Modules/mmapmodule.c

```diff
@@ -712,6 +766,7 @@ static struct PyMethodDef mmap_object_methods[] = {
     {"write_byte",      (PyCFunction) mmap_write_byte_method,   METH_VARARGS},
     {"__enter__",       (PyCFunction) mmap__enter__method,      METH_NOARGS},
     {"__exit__",        (PyCFunction) mmap__exit__method,       METH_VARARGS},
+    {"__repr__",        (PyCFunction) mmap__repr__method,       METH_NOARGS},
```

Comment:
Why is this being added here? `tp_repr` should be sufficient.

Reply:
Thanks. Just got it after doing some search. Will remove this.